commands/fun/country.js

```diff
@@ -0,0 +1,57 @@
+const Discord = require('discord.js');
+const config = require('../../configs/config.json');
+const fetch = require('node-fetch');
+const { MessageEmbed } = require('discord.js');
+
+module.exports = {
+    config: {
+        name: 'country',
+        description: '',
+        aliases: [""],
+        usage: '',
+        accessableby: "",
+    },
+    run: async (client, message, args) => {
+    const country = args.slice().join(' ');
+		if(!country) {
+			return message.channel.send(
+				':x: Please provide a valid Country.',
+			);
+		}
+		const url = 'https://restcountries.eu/rest/v2/name/' + country;
+
+		let response;
+		try {
+			response = await fetch(url).then(res => res.json());
+		}
+		catch (e) {
+			return message.channel.send(
+				':x:An error occured, please try again!',
+			);
+		}
+		try{
+			const data = response[0];
+			const embed = new MessageEmbed()
+				.setColor('BLUE')
```

Comment:
Can you replace the Color of the embed to the one in the `config` for all these commands?. As that's the one the bot uses globally.

Reply:
Done